tests/unit_tests/dft/include/compute_inplace_real_real.hpp

```diff
@@ -22,26 +22,26 @@
 
 #include "compute_tester.hpp"
 
-/* Test is not implemented because currently there are no available dft implementations.
- * These are stubs to make sure that dft::oneapi::mkl::unimplemented exception is thrown */
 template <oneapi::mkl::dft::precision precision, oneapi::mkl::dft::domain domain>
 int DFT_Test<precision, domain>::test_in_place_real_real_USM() {
     if (!init(MemoryAccessModel::usm)) {
         return test_skipped;
     }
+    if constexpr (domain == oneapi::mkl::dft::domain::REAL) {
+        // storage schemes for real transform arn't allowed to set the REAL_STORAGE to
+        // real_real or the COMPLEX_STORAGE to real_real as well. Skip.
```

Comment:
This comment is a little confusing. REAL_REAL is the only valid option for REAL_STORAGE and REAL-REAL storage is used in the valid branch of this test, this also applies to the out-of-place tests. I think you mean to say the real-complex transforms aren't supported.

Reply:
removed comment, from the looks of the oneMKL product its illegal to even set this, but I need to be sure, nonetheless removing the comment as it could be confusing